pages/README.md

```diff
@@ -1,6 +1,39 @@
-# PAGES
+# Lighthouse UI
```

Comment:
We've got a README and a pages/README ...

Reply:
This was done purposefully by Phil. Allows more fine grained approach for docs.